iron-scroll-target-behavior.html

```diff
@@ -75,8 +75,8 @@
         if (typeof scrollTarget === 'string') {
 
           var ownerRoot = Polymer.dom(this).getOwnerRoot();
-          this.scrollTarget = (ownerRoot && ownerRoot.$) ?
-              ownerRoot.$[scrollTarget] : Polymer.dom(this.ownerDocument).querySelector('#' + scrollTarget);
+          this.scrollTarget = (ownerRoot && ownerRoot.host && ownerRoot.host.$) ?
+              ownerRoot.host.$[scrollTarget] : Polymer.dom(this.ownerDocument).querySelector('#' + scrollTarget);
```

Comment:
Actually we can just use [domHost](https://github.com/Polymer/polymer/blob/master/src/mini/shady.html#L97) here,  e.g.
```
var host = this.domHost;
this.scrollTarget = (host && host.$) ?
    host.$[scrollTarget] : Polymer.dom(this.ownerDocument).querySelector('#' + scrollTarget);
```